app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    summa = sum(split_integer(17, 4))
+    assert summa == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
The test function `test_should_split_into_equal_parts_when_value_divisible_by_parts` is not correct. The function `split_integer(17, 4)` should return `[4, 4, 4, 5]` but in the assert statement you are comparing it with `[4, 4, 4, 4]`. The sum of the parts should be equal to the original number. This is a bug in the test.